packages/backend/src/controllers/authentication.ts

```diff
@@ -242,28 +389,9 @@ const genericOidcHandler =
         try {
             const { inviteCode } = req.session.oauth || {};
             req.session.oauth = {};
-            const [{ value: email }] = profile.emails;
-            const { id: subject } = profile;
-            if (!(email && subject)) {
-                return done(null, false, {
-                    message: 'Could not parse authentication token',
-                });
-            }
-            const [fallbackFirstName, fallbackLastName] = (
-                profile.displayName || ''
-            ).split(' ');
-            const openIdUser: OpenIdUser = {
-                openId: {
-                    issuer,
-                    email,
-                    subject,
-                    firstName: profile.name?.givenName || fallbackFirstName,
-                    lastName: profile.name?.familyName || fallbackLastName,
-                    issuerType,
-                },
-            };
+
             const user = await userService.loginWithOpenId(
-                openIdUser,
+                createOpenIdUser(undefined, profile, issuer, issuerType),
```

Comment:
yeah this call looks confusing (what is `undefined`?) - it should be `openIdUserFromPassportProfile`